fhir-model/src/main/java/com/ibm/fhir/model/util/FHIRUtil.java

```diff
@@ -433,7 +438,8 @@ public static String getExtensionStringValue(Resource resource, String extension
             if (DomainResource.class.isAssignableFrom(resource.getClass())) {
                 DomainResource dr = (DomainResource) resource;
                 for (Extension ext : dr.getExtension()) {
```

Comment:
Essentially 460 is duplicate - just a different origin.  Perhaps refactor into an intermediate method.

Reply:
done